src/js/gui.js

```diff
@@ -313,12 +313,9 @@ class GuiControl {
 
         this.switchery();
 
-        if (CONFIGURATOR.connectionValid) {
-            // Build link to in-use CF version documentation
-            const documentationButton = $('div#content #button-documentation');
-            documentationButton.html("Wiki");
-            documentationButton.attr("href", "https://github.com/betaflight/betaflight/wiki");
-        }
+        const documentationButton = $('div#content #button-documentation');
+        documentationButton.html("Wiki");
+        documentationButton.attr("href", "https://github.com/betaflight/betaflight/wiki");
```

Comment:
Have been working hours to forge a wiki page for firmware flasher. Now it just points to general wiki.
Please don't overwrite `href` for `GUI.active_tab === 'firmware_flasher`

Reply:
> Have been working hours to forge a wiki page for firmware flasher. Now it just points to general wiki. Please don't overwrite `href` for `GUI.active_tab === 'firmware_flasher`

well that code was there before me. I just removed the IF statement around it that prevents WIKI button to show normal.

Reply:
Best option would be to produce wiki pages for each different tab and refer as `wiki-url/${GUI.active_tab}`

But for now we have only this one, so we can make an temporary exception:

    if (GUI.active_tab !== 'firmware_flasher') {
        documentationButton.attr("href", "https://github.com/betaflight/betaflight/wiki");
    }

Reply:
We can revisit as we add other wiki pages

Reply:
oh i see, i didn't know the flasher tab already got its own wiki page link from Configurator.
Done